SQL/database_changelog.txt

```diff
@@ -0,0 +1,13 @@
+Any time you make a change to the schema files, remember to increment the database schema version. Generally increment the minor number, major should be reserved for significant changes to the schema. Both values go up to 255.
+
+The latest database version is 5.0; The query to update the schema revision table is:
+
+INSERT INTO `schema_revision` (`major`, `minor`) VALUES (5, 0);
+or
+INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (5, 0);
+
+In any query remember to add a prefix to the table names if you use one.
+
+----------------------------------------------------
+
+Version 5.0, 28 February 2018, by LaKiller8 - initial release, inspired by /tg/station schema
```

Comment:
You should probably start this at 1.0 unless you intend to keep parity with tgstation schema.

Reply:
Fair point, even though it saddens me a little I want us to differ in the end, I'll be keeping the TGDB relevant tables and columns the exact same while tweaking the statbus ones since we're a very different gamemode and someone will perhaps make our own statbus.